### ISSUE TITLE
x3dShininess fails

### ISSUE THREAD

**Issue by dnvasilev** [gearvrf/GearVRf-Tests#147]
Broken image:
![x3dshininess](https://user-images.githubusercontent.com/21369890/30298081-f103253a-96fe-11e7-9a36-21be307dcd7c.png)
Diff image (showing that we fail to render material and instead render some edge lines plus garbage)
![diff_x3dshininess](https://user-images.githubusercontent.com/21369890/30298143-203dfec4-96ff-11e7-86b0-a65f70e58e8b.png)

**Comment by mwitchwilliams** [comment]
I tested with gvr-x3d-demo.  A different texture map on the teapot, and inserted an X3D &lt;PointLight/&gt;
![torusteapot_wpointlight](https://user-images.githubusercontent.com/18128830/30337734-20a0f89a-979e-11e7-9eab-ae4df1136787.png)
Since this is an animation my screen shot won't match the master screen shot.

**Comment by mwitchwilliams** [comment]
I believe the Asset Test program may be moving all the objects into a centered location, including the point light.  As a result, the master image has the point light inside the teapot and torus, and the user sees just a silhouette image of the torus (as if it were backlit).

When I added this line:
`lightNode.getTransform().setPosition(0, 0, 100);`
to AssetLightTests.java :: x3dShininess(), to move the point light 100 units in front of the teapot and torus location, I got this image.
![x3dshininess](https://user-images.githubusercontent.com/18128830/30345305-6f767542-97b9-11e7-9e1f-d1266a7e183a.png)
